app.yaml

```diff
@@ -80,6 +80,10 @@ handlers:
   secure: always
   http_headers:
     Cache-Control: 'no-cache'
+- url: /extensions
+  static_dir: extensions
+  application_readable: true
```

Comment:
I'm slightly confused by why this needs to be application readable now and didn't before. Is it because we're doing file IO to read the files when interpolating the HTML for interactions that this needs to be application readable? Also to confirm my understanding, we need this section so that we're able to serve the interaction and validator JS files as <script> tags, is that correct?